src/footer/Footer.tsx

```diff
@@ -230,6 +231,10 @@ export default factory(function Footer({ middleware: { theme, i18n } }) {
 									v5.0
 									<img classes={themedCss.externalLink} alt="externalLink" src={externalLink} />
 								</a>
+								<div classes={themedCss.title}>{messages.languages}</div>
+								<a href="https://zh-CN.dojo.io" classes={css.link}>
```

Comment:
We should probably have a link for english here too for when they are on another language's version.